internal/ingress/controller/template/template_test.go

```diff
@@ -136,7 +136,7 @@ proxy_pass http://upstream-name;
 			"/",
 			`~* ^/something\/?(?<baseuri>.*)`,
 			`
-rewrite (?i)/something/(.*) /$1 break;
+rewrite (?i)/something(.*) /$1 break;
```

Comment:
These look a bit dangerous to me, the regexp should match either an end of line (`$`) or `/.*`, but not `/somethingelse` as it would be the case here.

Reply:
You are correct. I've changed it to only match on end of line or /.*

I'm still having trouble getting the tests to run without issues. The unit tests indicate FAIL, but without any specifics and the e2e tests I've not been able to run successfully at all yet. I did however test an image build from my fork/branch, tested it in our own environment and there it seems to do the right thing.